stories/off-screen/off-screen.stories.js

```diff
@@ -0,0 +1,22 @@
+export default {
+    title: 'Components/Off Screen',
+    parameters: {
+        description: `For aria-live to read any change in element, we need to place element but it should not be visible to user.
+            same time, it can not be made hidden or display: none, as screen reader will not be able to pick this element.
+            This component will make sure that element is not visible on screen but it can be read by screen reader.`,
```

Comment:
```suggestion
        description: `For aria-live to read any change in element, we need to place element but it should not be visible to user.
            At the same time, it can not be made \`visibility: hidden\` or \`display: none\`, as screen reader will not be able to pick this element.
            This component will make sure that element is not visible on screen but it can be read by screen reader.`,
```

Reply:
fixed